src/network.rs

```diff
@@ -87,10 +112,18 @@ pub fn create_swarm() -> Result<Swarm<StoryBehaviour>, Box<dyn std::error::Error
         .multiplex(yamux::Config::default())
         .boxed();
 
+    // Create request-response protocol for direct messaging
```

Comment:
The request-response protocol configuration uses default settings. Consider adding timeouts and retry policies for better reliability.